.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     # 3. Commit this change to the PR branch where the changes exist.
     current_golden_images_hash:
         type: string
-        default: 87f8ef3c150ec78b2bd9d21c0d101eb050b6eed1
+        default: 2d02ebdacf1733388ccfd07c031c217eeb533160
```

Comment:
VRT diffs were expected as focus should not be set on menus programmatically opened.